### PR TITLE
mc: new package at 4.8.23

### PR DESCRIPTION
**Pull request by penguin-ff**
## Description of package

GNU Midnight Commander is a visual file manager with a feature rich full-screen text mode that allows you to copy, move and delete files and whole directory trees, search for files and run commands in the subshell. Internal viewer and editor are included

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/mc/sources [hunk 1]
```diff
@@ -0,0 +1 @@
+http://ftp.midnight-commander.org/mc-4.8.23.tar.xz
```
Comment (dylanaraps) [inline community/mc/sources]: Is there an `https` source available?

Comment (penguin-ff) [reply community/mc/sources]: Found one - see my last commit

**Comment by dylanaraps** [comment]
Is the new source an official mirror of `mc` or something third-party?

**Comment by penguin-ff** [comment]
The former http url was i think the "official" one found on their website and also used by gentoo and arch. The new https url is "stolen" from the freebsd port. The checksums are the same (see commit)

**Comment by dylanaraps** [comment]
If the `mc` developers don't provide a `https` mirror I'd much rather we stick to their official source even if it is `http`. I think this is an issue we should raise upstream, apologies for the run around.  :+1:

**Comment by penguin-ff** [comment]
No problem. 
There is also a https link to github on their site, but this tarball has no configure script. You have the whole automake / autoconf mess and i think one need also autopoint from the gettext package (not packaged yet in kisslinux i think)
Is the pull request now ready to merge for you?

**Comment by dylanaraps** [comment]
Swap it back to the original `http` link and I'll merge. :+1:

**Comment by dylanaraps** [comment]
LGTM :+1:

**Pull request merged**